vscode-extensions.json

```diff
@@ -65,12 +65,12 @@
     },
     {
       "repository": "https://github.com/camel-tooling/vscode-camelk",
-      "revision": "0.0.18",
+      "revision": "0.0.19",
       "sidecar": {
-        "image": "quay.io/eclipse/che-sidecar-camelk:0.0.18-39f19dd",
+        "image": "quay.io/eclipse/che-plugin-sidecar:camelk-2d33352",
         "source": {
-          "repository": "https://github.com/che-dockerfiles/che-sidecar-camelk"
-        }
+          "repository": "https://github.com/eclipse/che-plugin-registry"
```

Comment:
tried to guess

Reply:
it's ok for this file as it'll removed after https://github.com/eclipse/che-plugin-registry/pull/732 is merged